docs/index.md

```diff
@@ -8,6 +8,14 @@ meta-description: Learn how to install, integrate, configure and develop CKEdito
 
 # CKEditor 4 Documentation
 
+<info-box hint>
```

Comment:
Shouldn't this be a warning/alert type of infobox?

Reply:
Warning maybe, I wouldn't got for a red infobox. I'll test it.